modules/virt-enlarging-vm-disk-pvc.adoc

```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc
+// * virt/virtual_machines/virtual_disks/virt-expanding-vm-disks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-enlarging-vm-disk_{context}"]
-= Enlarging a virtual machine disk
+[id="virt-enlarging-vm-disk-pvc_{context}"]
+= Expanding a PVC
 
-Virtual machine (VM) disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+You can increase the size of a virtual machine (VM) disk by expanding the persistent volume claim (PVC) of the disk.
```

Comment:
`You can increase the size of a virtual machine (VM) disk by expanding the persistent volume claim (PVC) of the disk, or by adding an additional virtual disk`